### ISSUE TITLE
flooding error log with "window.queueMicrotask is not a function"

### ISSUE THREAD

**Issue by ktownsend-personal** [thomasloven/hass-browser_mod#157]
My Home Assistant version: 0.116.4

What I am doing:
Nothing

What I expected to happen:
No errors logged

What happened instead:
Every 10 seconds I get 3 errors logged. I narrowed it down to my 3 Amazon Fire tablets. Possibly related to being on WebView version 70, which I cannot update due to a limitation of Fire OS having the webview pointer in the firmware.
```
ERROR (MainThread) [frontend.js.latest.202010012] http://192.168.34.13:8123/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
```

**Minimal** steps to reproduce:
browser-mod latest version installed (1.2.3)

Error messages from the browser console:
No errors in Chrome on PC. Unable to check on Fire tablets.

---

**By putting an X in the boxes ([ ]) below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by thomasloven** [comment]
Well.. that's anoying. It's supported from Webview 71 (November 2018)...

**Comment by KTibow** [comment]
Any way to do a fallback function?

**Comment by thomasloven** [comment]
I guess adding this line to the top of `custom_components/browser_mod/brower_mod.js` would do it.
```js
window.queueMicrotask = window.queueMicrotask | (f) => Promise.resolve().then(f);
```

I'll have a fix out soon.

**Comment by ktownsend-personal** [comment]
@thomasloven Thanks!

**Comment by ktownsend-personal** [comment]
I got errors about `Unexpected token .` at line 1 position 31 when I tried your line of code at the top of the file, but it seems to be ok with this adjustment:
```js
if (!window.queueMicrotask) window.queueMicrotask = (f) => Promise.resolve().then(f);
```

**Comment by Jpsy** [comment]
I witnessed this problem on an Amazon Fire Tablet.
It is the result of an outdated Webview component.
Android Webview supports queueMicrotask since v0.71. My Fire was running v0.70.

I did two things to solve the problem:

I updated Webview through Google Play Store. (I have manually installed Play Store on my Fire Tablet).
I updated Fire OS from v7.3.1.4 to the latest v7.3.1.6.
I now have Webview v0.75 and the problem is gone.
